products/focus/testing_focus.md

```diff
@@ -10,10 +10,11 @@ Screenshots are provided by the mobile teams and sent out on dev-l10n. The lates
 
 ## Test builds
 
-While there are testing builds available via BuddyBuilds, the process is currently manual and localizers should reach out to the l10n mobile project manager in order to be added to these builds (currently **delphine** at **mozilla** dot **com**).<br/>
+For Focus Android, while there are Nightly builds available via the Google Play Store, the process is currently manual and localizers should reach out to the l10n mobile project manager in order to be added to these builds (currently **delphine** at **mozilla** dot **com**), and send out the Google email that they use to connect to Google Play Store. Once the account has been whitelisted (usually the next day), localizers will need to opt-in with the same Google account by going [to this URL](https://play.google.com/apps/internaltest/4698596548865727179). At the next update, the app will update to the Nightly version.
```

Comment:
> and send out the Google email

Maybe "and include in the request the email that they use…".
I'm also not sure it has to be a Google email?

> At the next update, the app will update to the Nightly version.

I think this is true only if the app is already installed, otherwise it's a normal app installation (the latter was the case for me).

Reply:
Yes it has to be a Google email.
But you made me remember something... There should still be updated builds on Taskcluster for those who don't want to use Google Play. So I'll update with that info as well!